CHANGELOG.asciidoc

```diff
@@ -0,0 +1 @@
+This is a placeholder file. Release notes are published in CHANGELOG.md
```

Comment:
Note that this file will be overwritten by #557. It is added here because the file must exist before elastic/docs#1275 can be merged (which in turn must exist before #557 can be merged).